packages/core/src/odata/common/filter/filterable.ts

```diff
@@ -67,3 +72,40 @@ export function or<EntityT extends EntityBase>(
 // Export function not<EntityT extends Entity>(expression: Filterable<EntityT>): Filterable<EntityT> {
 //   Return new FilterList([], expressions);
 // }
+
```

Comment:
Is the todo above still relevant? Perhaps we should include a linter check warning for commented out code?

Reply:
https://github.wdf.sap.corp/MA/scp-rocket/issues/1270 for all todos